packages/nodes-base/credentials/Aws.credentials.ts

```diff
@@ -31,6 +31,16 @@ export class Aws implements ICredentialType {
 				password: true,
 			},
 		},
+		{
+			displayName: 'Session Token',
+			name: 'sessionToken',
+			description: '(Optional) Temporary credentials from AWS STS',
```

Comment:
Change description to: "This is an optional field. If you are not using temporary credentials, leave it blank."

Reply:
Done!